exercises/BatchSend.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.4.24;
 contract BatchSend {
     event BatchSent(address[] addresses);
 
-//create a constructor function here with following
+//create a function here with following
```

Comment:
Don't know that we need these comments if we are going to the leave the function defined for them.  If we remove the params and modifiers from the method signature than I think it makes sense.  I'm fine with either, 1. remove comments and leave method or 2. leave comment and remove params and modifiers from method.